pkg/common/cns-lib/vsphere/virtualcentermanager.go

```diff
@@ -52,6 +54,8 @@ type VirtualCenterManager interface {
 	IsvSANFileServicesSupported(ctx context.Context, host string) (bool, error)
 	// IsExtendVolumeSupported checks if extend volume is supported or not.
 	IsExtendVolumeSupported(ctx context.Context, host string) (bool, error)
+	// IsOnlineExtendVolumeSupported checks if online extend volume is supported or not wrt vcenter versions
```

Comment:
`IsOnlineExtendVolumeSupported checks if online extend volume is supported or not wrt vcenter versions` => `IsOnlineExtendVolumeSupported checks if online extend volume is supported or not on the Center Host`

Reply:
Changed it